app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def show; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+  end
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍